bigquery/tables.js

```diff
@@ -34,7 +25,127 @@ var Storage = require('@google-cloud/storage');
+  var config = {};
+  if (options.schema) {
+    config.schema = options.schema;
+  }
+
+  // See https://googlecloudplatform.github.io/gcloud-node/#/docs/bigquery/latest/bigquery/table
+  // table.create(config, function (err, table) {
+  dataset.createTable(options.table, config, function (err, table) {
+    if (err) {
+      return callback(err);
+    }
+
+    console.log('Created table: %s', options.table);
+    return callback(null, table);
+  });
+}
+// [END create_table]
+
+// [START list_tables]
+/**
+ * List tables in the specified dataset.
+ *
+ * @param {object} options Configuration options.
+ * @param {string} options.dataset The dataset of the new table.
+ * @param {Function} callback Callback function.
+ */
+function listTables (options, callback) {
+  var dataset = bigquery.dataset(options.dataset);
+
+  // See https://googlecloudplatform.github.io/gcloud-node/#/docs/bigquery/latest/bigquery/dataset
+  dataset.getTables(function (err, tables) {
+    if (err) {
+      return callback(err);
+    }
+
+    console.log('Found %d table(s)!', tables.length);
+    return callback(null, tables);
+  });
+}
+// [END list_tables]
+
+// [START delete_table]
+/**
+ * Creates a new table with the given name in the specified dataset.
+ *
+ * @param {object} options Configuration options.
+ * @param {string} options.dataset The dataset of the new table.
+ * @param {string} options.table The name for the new table.
+ * @param {function} cb The callback function.
+ */
+function deleteTable (options, callback) {
+  var table = bigquery.dataset(options.dataset).table(options.table);
+
+  // See https://googlecloudplatform.github.io/gcloud-node/#/docs/bigquery/latest/bigquery/table
+  table.delete(function (err) {
+    if (err) {
+      return callback(err);
+    }
+
+    console.log('Deleted table: %s', options.table);
+    return callback(null);
+  });
+}
+// [END delete_table]
+
+// [START import_file]
+/**
+ * Load a csv file into a BigQuery table.
+ *
+ * @param {string} file Path to file to load.
+ * @param {string} dataset The dataset.
+ * @param {string} table The table.
+ * @param {string} [format] The format of the file to be imported.
+ * @param {function} callback The callback function.
+ */
+function importFile (options, callback) {
+  var file;
+  if (options.bucket) {
+    // File is in Google Cloud Storage, e.g. gs://my-bucket/file.csv
+    file = storage.bucket(options.bucket).file(options.file);
+  } else {
+    // File is local, e.g. ./data/file.csv
+    file = options.file;
+  }
+  var table = bigquery.dataset(options.dataset).table(options.table);
+  var config = {
+    format: options.format
+  };
+
+  // See https://googlecloudplatform.github.io/gcloud-node/#/docs/bigquery/latest/bigquery/table?method=import
+  table.import(file, config, function (err, job) {
+    if (err) {
+      console.log(err.stack);
+      return callback(err);
+    }
+
+    console.log('Started job: %s', job.id);
```

Comment:
Nah, I think the user gets that information when they pick which function to call in the CLI, e.g. `node tables import ...`